packages/core/src/App/Components/Layout/Footer/complaints-policy.jsx

```diff
@@ -0,0 +1,243 @@
+                        text: localize(
+                            'We shall acknowledge receiving your complaint, review it carefully, and keep you updated on the handling process. We might request further information or clarifications to facilitate the resolution of the complaint.'
+                        ),
+                    },
+                    {
+                        title: localize('3.3. Resolving your complaint'),
+                        text: localize(
+                            'We shall try to resolve your complaint within 15 business days. We will inform you of the outcome together with an explanation of our position and propose any remedial measures we intend to take.'
+                        ),
+                    },
+                ],
+            },
+        },
+        {
+            title: localize('4. Your decision'),
+            content: {
+                text: (
+                    <Localize
+                        i18n_default_text='If you are not satisfied with the outcome, you can escalate your complaint to the <0>Financial Commission</0>.'
+                        components={[
+                            <a
+                                key={0}
+                                className='link link--orange'
+                                rel='noopener noreferrer'
+                                target='_blank'
+                                href='https://financialcommission.org/'
+                            />,
+                        ]}
+                    />
+                ),
+            },
+        },
+    ];
+
+    const complaints_procedure = [
+        {
+            title: localize('Financial commission complaints procedure'),
+        },
+        {
+            title: localize('1. Filing complaints'),
+            list: [
+                <Localize
+                    key={0}
+                    i18n_default_text='<0>a.</0>You may file a complaint with the Financial Commission up to 45 days after the incident.'
+                    components={[<span className='complaints-policy__list-item-prefix' key={0} />]}
+                />,
+                <Localize
+                    key={1}
+                    i18n_default_text='<0>b.</0>The Financial Commission has 5 days to acknowledge that your complaint was received and 14 days to answer the complaint through our Internal Dispute Resolution (IDR) procedure.'
+                    components={[<span className='complaints-policy__list-item-prefix' key={0} />]}
+                />,
+                <Localize
+                    key={2}
+                    i18n_default_text='<0>c.</0>You will be able to file a complaint with the Financial Commission only if you are not satisfied with our decision or the decision wasn’t made within 14 days.'
+                    components={[<span className='complaints-policy__list-item-prefix' key={0} />]}
+                />,
+            ],
```

Comment:
Suggestion: Is it possible to use `list-style-type: lower-alpha` instead of using `span`? 
```html
<ol>
  <li>You may file a complaint with the Financial Commission up to 45 days after the incident. </li>
</ol>
```
It might be easier to translate.

Reply:
so, we don't need to translate the prefix?